docs/Documentation/Scripting/Building-Blocks/Objectives-List.md

```diff
@@ -39,14 +39,17 @@ give accurate results. Experiment with this objective a bit to make sure you've
+|------------------|------------------------------------------------------|--------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| _Block Type_     | [Block Selector](../Data-Formats.md#block-selectors) | :octicons-x-circle-16:   | The block which must be broken / placed.                                                                                                                                                                                                                                  |
+| _Amount_         | Number                                               | :octicons-x-circle-16:   | The amount of blocks to break / place. Less than 0 for breaking and more than 0 for placing blocks.                                                                                                                                                                       |
+| _Safety Check_   | Keyword (`noSafety`)                                 | Safety Check Enabled     | The Safety Check prevents faking the objective. The progress will be reduced when the player does to opposite of what they are supposed to do. Example: Player must break 10 blocks. They place 10 of their stored blocks. Now the total amount of blocks to break is 20. |
+| _Notifications_  | Keyword (`notify`)                                   | Disabled                 | Displays messages to the player each time they progress the objective. Optionally with the notification interval after colon.                                                                                                                                             |
+| _Location_       | loc:location                                         | Optional. Default: none  | Adds an optional location to the objective, only counting blocks broken/placed at the specific location.                                                                                                                                                                  |
+| _Region Definer_ | region:location                                      | Optional. Default: none  | Adds an optional second location to only count blocks broken/placed in a rectangle between the specified location and this location. This won't have an effect if parameter location isn't set.                                                                           |
+| _ignorecancel_   | Keyword (`ignorecancel`)                             | Objective won't progress | Allows the objective to progress, even if the event is cancelled by the Server. For example if the player is not allowed to build.                                                                                                                                        |
```

Comment:
A very small detail: 
> Objective won't progress

might be a bit confusing for users, maybe change to "protected blocks will not affect the objective" or something similar

```suggestion
| _ignorecancel_   | Keyword (`ignorecancel`)                             | Objective won't progress | Allows the objective to progress, even if the event is cancelled by the Server. For example if the player is not allowed to build.                                                                                                                                        |
```